wls-gui-wahllokalsystem/tests/composables/broadcast/broadcastService.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { useBroadcastTestDataFactory } from "@tests/types/broadcast/BroadcastTestDataFactory.ts";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+
+import { useBroadcastService } from "@/composables/broadcast/broadcastService.ts";
+
+const mockDefinitions = vi.hoisted(() => ({
+  mapDtoToModel: vi.fn(),
+  addNotification: vi.fn(),
+  configurationConstructor: vi.fn().mockImplementation(() => ({})),
+  deleteMessage: vi.fn(),
+  getMessage: vi.fn(),
+}));
+
+vi.mock("@/api/wls-clients/generated-broadcast-api", () => ({
+  BroadcastControllerApi: vi.fn().mockImplementation(() => ({
+    getMessage: mockDefinitions.getMessage,
+    deleteMessage: mockDefinitions.deleteMessage,
+  })),
+  Configuration: mockDefinitions.configurationConstructor,
+}));
+vi.mock("@/composables/broadcast/broadcastMapper.ts", () => ({
+  useBroadcastMapper: () => ({
+    dtoToModel: mockDefinitions.mapDtoToModel,
+  }),
+}));
+vi.mock("@/composables/userNotification/userNotificationService.ts", () => ({
+  useUserNotificationService: () => ({
+    addNotification: mockDefinitions.addNotification,
+  }),
+}));
+
+const { createBroadcastMessage, createMessageDTO } =
+  useBroadcastTestDataFactory();
+
+describe("BroadcastService.ts", () => {
+  const { getMessage, deleteMessage } = useBroadcastService();
+
+  beforeEach(() => {
+    vi.resetAllMocks();
+    vi.clearAllMocks();
+  });
+
+  describe("getMessage", () => {
+    it("should_returnBroadcastMessage_when_messageIsReceivedFromApi", async () => {
+      const wahlbezirkID = "wahlbezirkID";
+
+      const mockedApiResponseBody = createMessageDTO();
+      mockDefinitions.getMessage.mockReturnValue(
+        Promise.resolve({ status: 200, data: mockedApiResponseBody })
+      );
```

Comment:
```suggestion
      mockDefinitions.getMessage.mockReturnValue(
        Promise.resolve({ status: 200, data: createMessageDTO() })
      );
```
`mockedApiResponseBody` wird nirgendwo anders gebraucht deswegen denke ich ist eine extra variable nicht notwendig